### PR TITLE
"How to Register a Dataset with Wikidata" ready for review

### PR DESCRIPTION
**Pull request by egonw**
This recipe describes how to list a dataset with DOI in Wikidata so that it can be used as reference in Wikidata. A snapshot is available from the attached PDF.

[Registering Datasets.pdf](https://github.com/FAIRplus/the-fair-cookbook/files/7212728/Registering.Datasets.pdf)

### CODE REVIEW

File: content/recipes/findability/registeringDatasets.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+To register datasets to [Wikidata](https://www.wikidata.org/) requires a Wikidata account ([create one](https://www.wikidata.org/wiki/Special:CreateAccount)).
+
+---
+
+
+## Main Content
+
+This recipes takes an interest in datasets with information about small chemical compounds, particularly datasets with an SDF file or spreadsheet with SMILES.
+But the registration itself in Wikidata to make it more findable works for any dataset. The registration in Wikidata makes the metadata more findable by
+itself {footcite}`Waagmeester2020`. This [Wikidata:Introduction](https://www.wikidata.org/wiki/Wikidata:Introduction) describes the principle ideas and data model that will make this
```
[the remark this answers is not in the document]
Comment (egonw) [reply content/recipes/findability/registeringDatasets.md:45]: done.

File: content/recipes/findability/registeringDatasets.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+recipe easier to follow.
+
+Importantly, Wikidata allows the dataset to be used as reference in Wikidata to support statements. For example, if the dataset contains a logP value,
+then this logP value can be associated to the compound in Wikidata with a statement, with the dataset itself as reference.
```
[the remark this answers is not in the document]
Comment (egonw) [reply content/recipes/findability/registeringDatasets.md:49]: Applied

File: content/recipes/findability/registeringDatasets.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+### Finding Datasets
+
+There are several ways to locate datasets to be registered into Wikidata. One consideration is that the dataset has the recommended license and an identifier.
+
+Example of where open datasets with information about chemical compounds can be found include:
+
+* [Figshare](https://figshare.com/)
+* [Zenodo](https://zenodo.org/)
+* [Joint Research Centre Data Catalogue](https://data.jrc.ec.europa.eu/dataset)
+
```
[the remark this answers is not in the document]
Comment (egonw) [reply content/recipes/findability/registeringDatasets.md:61]: Better explained at the start of the recipe now.

File: content/recipes/findability/registeringDatasets.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+itself {footcite}`Waagmeester2020`. This [Wikidata:Introduction](https://www.wikidata.org/wiki/Wikidata:Introduction) describes the principle ideas and data model that will make this
+recipe easier to follow.
+
+Importantly, Wikidata allows the dataset to be used as reference in Wikidata to support statements. For example, if the dataset contains a logP value,
+then this logP value can be associated to the compound in Wikidata with a statement, with the dataset itself as reference.
+<!-- TODO: add screenshot of example -->
+
+### Finding Datasets
+
+There are several ways to locate datasets to be registered into Wikidata. One consideration is that the dataset has the recommended license and an identifier.
```
[the remark this answers is not in the document]
Comment (egonw) [reply content/recipes/findability/registeringDatasets.md:54]: Applied.

File: content/recipes/findability/registeringDatasets.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+TODO: add below statement when that recipe is included too:
+
+For details related to finding datasets and determining the correct licenses, please refer to Finding Compounds. 
+-->
+
+### Dataset Registration in Wikidata
+
+After locating a dataset, check the availability of the details of the dataset. Essential attributes include the
+DOI, the title of the dataset, the author(s) of the dataset, publication date of the dataset, and links to the
+dataset source. You may need this later.
```
[the remark this answers is not in the document]
Comment (egonw) [reply content/recipes/findability/registeringDatasets.md:80]: Applied.

File: content/recipes/findability/registeringDatasets.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+
+With the DOI, there is a good chance that Scholia {footcite}`Nielsen2017` can help you register the dataset in Wikidata.
```
[the remark this answers is not in the document]
Comment (egonw) [reply content/recipes/findability/registeringDatasets.md:82]: Applied.

File: content/recipes/findability/registeringDatasets.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+You can use the `https://scholia.toolforge.org/doi/$DOI` pattern ('$DOI' in the link must be replaced with the
+DOI of the dataset you are registering) to check if your dataset is already listed in
+Wikidata. If not this page will use the DOI to convert the metadata transformed into a Quickstatements.
```
[the remark this answers is not in the document]
Comment (egonw) [reply content/recipes/findability/registeringDatasets.md:85]: Applied.

**Comment by egonw** [comment]
@proccaserra:

> @egonw , thx for the PR. Please find my comments and corrections . I guess the main comments I have about the recipe are as follows:

Thanks. Sorry it took me longer to reply than I had in mind.

> * a justification for why submitting the Wikidata if a dataset is already registered in Zenodo? I feel the benefits of registering into Wikidata should be expanded or explained

I added some words about this.

> * It would be nice to have a few words about `Scholia Quickstatement` tool

Done.

> * create a table of tools used in the recipe (Wikidata, Scholia Quickstatements)

Do you have a recipe with an example table like that?

> * Also, since you write that datasets of particular interest are those containing SMILES strings, is there a mechanism in Scholia Quickstatements to pull those if found in a dataset ?

No, but one of the other recipes is about this (and project with Robert/Zachary in general).

> * Finally, how would you rate the Maturity of such Capability? @iemam could you have a look at this too ?

I'm going to need a bit more guidance here. If you mean the maturity of this recipe? It's been in action for multiple months already, tho mostly used for articles, not datasets.

**Comment by egonw** [comment]
@proccaserra, I checked, but I applied the changes I could apply. For other things I need more guidance, see my earlier reply.

**Comment by proccaserra** [comment]
@egonw thx for the corrections. Regarding the Maturity Indicator rating, we'll work with @tabbassidaloii and @iemam on this and get back to you if anything specific arises

**Comment by egonw** [comment]
@proccaserra, has this work now been merged? Because I cannot find it on the deployed website.

**Comment by proccaserra** [comment]
@egonw I have now merged but I noticed there are a couple of TODOs remaining.
One is for me but the other one needs your input : -> (logP related in the intro: 

thx

**Comment by egonw** [comment]
yeah, it's a nice to have: I do not have a registered data set with logP data yet to which I can add 'logp' as 'main subject'.

The other one has to wait for the next recipe to be done indeed.

**Pull request merged**